configure-broker-to-use-a-different-channel.md

```diff
@@ -0,0 +1,286 @@
+      name: gcp-pubsub
+END
+```
+
+The one major catch with this approach is if you are manipulating the `Broker`
+named `default` and the [namespace is annotated](todo) to ensure the `Broker` named
+`default` exists. In all likelihood, your `create` command will not happen until
+after the reconciler has created one already. So we can do either of the following:
+- [Remove the annotation](#remove-the-annotation)
+- [Change the Channel inflight](#change-the-channel-inflight)
+
+#### Remove the annotation
+
+We will remove the annotation, make our changes, then re-add the annotation.
+
+```shell
+# Disable the annotation.
+kubectl label namespace $BROKER_NS --overwrite 'eventing.knative.dev/injection=off
+
+# Delete the existing Broker.
+kubectl -n $BROKER_NS delete broker $BROKER_NAME
+
+# Create the new Broker.
+kubectl -n $BROKER_NS create -f - << END
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: $BROKER_NAME
+spec:
+  channelTemplate:
+    provisioner:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: ClusterChannelProvisioner
+      name: gcp-pubsub
+END
+
+# Enable the annotation.
+kubectl label namespace $BROKER_NS --overwrite 'eventing.knative.dev/injection=enabled
+```
+
+#### Change the Channel Inflight
+
+**Dangerous**
+
+Forcing the `Channel`s of an exsiting `Broker` to change is **dangerous** it will
+drop all events that are currently inflight, along with new events until the control
+and data planes are healthy again.
+
+If you want to do this, then the procedure is to first update the `Channel` that
+the `Broker` will [theoretically use](#which-channel-is-my-broker-_theoretically_-using).
+Most likely via setting its `spec.channelTemplate`.
+
+```shell
+kubectl -n $BROKER_NS apply -f - << END
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: $BROKER_NAME
+spec:
+  channelTemplate:
+    provisioner:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: ClusterChannelProvisioner
+      name: gcp-pubsub
+END
+```
+
+Then manually delete the existing `Channel`s:
+
+```shell
+kubectl -n $BROKER_NS delete channels -l "eventing.knative.dev/broker=$BROKER_NAME" 
+```
+
+That should delete the existing `Channel`s. Replacement `Channel`s will be
+created very quickly by by the `Broker` reconciler.
+
+```shell
+kubectl -n $BROKER_NS get channels -l "eventing.knative.dev/broker=$BROKER_NAME"
+```
+
+Which will return something like:
+
+```shell
+NAME                   READY   REASON   AGE
+default-broker-krv99   True             5s
+default-broker-zjrjd   True             5s
+```
+
+Note the `AGE` column says they are very young (five seconds in this example).
+
+Verify that they have the correct provisioner:
+
+```shell
+kubectl -n $BROKER_NS get channels -l "eventing.knative.dev/broker=$BROKER_NAME" -o jsonpath='{.items[*].spec.provisioner.name}'
+```
+
+The output should look something like:
+
+```shell
+gcp-pubsub gcp-pubsub
```

Comment:
1. Having a section with a link to a page that lists all available channels will help
 - That page will have all channels listed with brief description and link to each channel's readme file
2. Next steps
 - Introduction to Event Registry
3. Further reading:
 - How to write a custom channel.
 - Link to Channel, Subscription, and Operator model concepts.